src/test/resources/features/highlighting/inspections/duplicated_args_inspection.feature

```diff
@@ -9,20 +9,25 @@ Feature: Inspection for duplicated arguments in same section
     And SmkSectionDuplicatedArgsInspection inspection is enabled
     Then I expect inspection warning on <"target_2"> in <"target_2" # duplicate> with message
     """
-    This argument has already been added to '<section>' section.
+    This argument has been already added to '<section>' section.
     """
     When I check highlighting warnings
     Examples:
-      | section              | rule_like  |
-      | input                | checkpoint |
-      | input                | rule       |
-      | output               | rule       |
-      | params               | rule       |
-      | resources            | rule       |
-      | log                  | rule       |
-      | wildcard_constraints | rule       |
+      | section              | rule_like   |
```

Comment:
In our other tests `rule_like` column is usually before `section` column. Could you please swap them here for consistency. You could use a column selection mode feature in IDEA to cut the whole column and past it in the right place (see https://www.jetbrains.com/help/rider/Multicursor.html#column_selection).

Reply:
Done in d3530b55a64b2c25d87b8154f14ca225a634a17a